iam/api-client/src/main/java/com/google/iam/snippets/GrantableRoles.java

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 Google Inc.
```

Comment:
Please use "LLC", as well as javadoc style comments in this and all other java files, i.e.

/** Copyright 2018 Google LLC
 *
 * Licensed...
 */

Reply:
Done -- however, this now causes the tests to fail: 

RegexpHeader: Line does not match expected header line of '^\W+\*?\W*Copyright 20\d\d Google Inc.$'.

Seems like the test needs to be updated to use LLC...